package.json

```diff
@@ -39,6 +41,7 @@
     "babel-eslint": "8.2.3",
     "babel-loader": "7.1.4",
     "babel-preset-env": "1.7.0",
+    "babel-preset-flow": "6.23.0",
```

Comment:
let's leave this out for later. it strips type information, which we don't necessarily want to do.

Reply:
okay.. Cool. I'll remove this.